Test rearrangement (#429)
mozilla-ai/lumigator: lumigator/python/mzai/backend/README.md

## Test instructions

> The backend includes both unit tests (requiring no additional containers) and integration tests (currently requiring a live Ray server in the same server where the tests are running, and test LocalStack containers started up in a fixture within tests).

Reviewer: ```suggestion
The backend includes both unit tests (requiring no additional containers) and integration tests (currently requiring a live Ray instance in the same network where the application and tests are running, and test LocalStack containers also in the same configuration.
```

Author: Adding a ) at the end ^_^

Note that this will create an SQLite database file named `local.db` in the `backend` directory. Remove it before running another batch of tests.

> The tests have been separated into a unit test suite and an integration test suite, according to the external containers needed for each run. There are make targets available at the root folder, as follows:

Reviewer: ```suggestion 
The tests include a unit test suite and an integration test suite. There are make targets available at the root folder, as follows:
```

* `backend-test`: runs `backend-unit-test` and `backend-int-test`
  * `backend-unit-test`: runs tests in `backend/tests/unit/*/test_*.py` (any depth of subfolders)
  * `backend-int-test`: runs tests in `backend/tests/int/*/test_*.py` (any depth of subfolders)

> The SQLite configuration and the test containers are started up appropriately.

Reviewer: ```suggestion
The SQLite configuration and the test containers are started for integration tests.
```
